.lgtm.yml

```diff
@@ -0,0 +1,9 @@
+path_classifiers:
+  library:
+    - tools
+  generated:
+    # The exports defined in __init__.py are defined in the Cython module
+    # np.random.mtrand. By excluding this file we suppress a number of
+    # "undefined export" alerts
```

Comment:
Shouldn't we only be doing this for `random/__init__.py`?

Reply:
Good catch, @eric-wieser! Thanks — fixed it.